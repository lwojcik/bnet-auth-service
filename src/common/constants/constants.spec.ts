import * as constants from './index';

describe('constants', () => {
  it('should be defined', () => {
    expect(constants).toMatchSnapshot();
  });
});
