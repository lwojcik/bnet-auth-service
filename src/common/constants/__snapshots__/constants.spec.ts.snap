// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants should be defined 1`] = `
Object {
  "APP_INFO": Object {
    "appPrefix": "BAS",
    "description": "REST microservice for retrieving and caching OAuth access tokens from Blizzard Battle.net API",
    "name": "bnet-auth-service",
  },
}
`;
